app/controllers/application_controller.rb

```diff
@@ -166,6 +167,14 @@ def set_selected_branding_group
     end
   end
 
+  # Kludge to fix problem with signed-in users not having accepted Terra Terms of Service
```

Comment:
NIT - according to google (since I had never heard of this word) "Kludge" is a slang word. Perhaps we should refrain from using a word with an informal definition here for better understanding in the future.

Reply:
[Oxford English Dictionary defines "kludge"](https://www.oxfordlearnersdictionaries.com/us/definition/american_english/kludge) without qualifying it as slang, so it seems reasonably formally defined.  Wikipedia [notes](https://en.wikipedia.org/wiki/Kludge) the term is "used in diverse fields such as computer science, aerospace engineering, Internet slang, evolutionary neuroscience, and government".

Ironically enough, I opted for "kludge" over "hack" hoping to avoid word choice issues.  "Kludge" is fairly well-known and more succinct than saying e.g. "temporary sub-optimal fix".  And unlike "hack", the word "kludge" isn't confusable with "maliciously exceed authorized access".

That said, I've taken your suggestion!  See 7117823.  I also documented the planned refinement.